DataPerProcessing/utilities.py

```diff
@@ -71,13 +97,21 @@ def getBBoxList(filepath):
 	* filepath: Path of label file
 	'''
 	ObjectCoordinatesList = []
+	ObjectNamesList = []
+	ObjectDifficultyList = []
 	with open(filepath, 'r') as labelFile:
 		ObjectList = labelFile.readlines()
 	for Object in ObjectList:
-		ObjectCoordinatesList.append([int(float(i)) for i in Object.split(' ')[:8]])
-	return ObjectCoordinatesList
```

Comment:
wrong return

Reply:
Why?
The function is suppose to return ObjectCoordinateList as a 2D list with all the bbox.